packages/server/src/mongo-admin/index.js

```diff
@@ -45,14 +48,32 @@ async function _createIndexes() {
   userCol.createIndex({ kusamaAddress: 1 }, { unique: true, sparse: true });
   userCol.createIndex({ polkadotAddress: 1 }, { unique: true, sparse: true });
 
-  reactionCol.createIndex({ commentId: 1, userId: 1 }, { unique: true });
+  const exists = await reactionCol.indexExists("commentId_1_userId_1")
+  if (exists) {
+    await reactionCol.dropIndex("commentId_1_userId_1");
+  }
+  reactionCol.createIndex({ commentId: 1, userId: 1 }, {
```

Comment:
Currently we don't need the reaction for grading, could you remove all the related code for grading reactions?